adagucserverEC/CConvertKNMIH5VolScan.cpp

```diff
@@ -28,7 +28,7 @@
 #include "COGCDims.h"
 #include "CCDFHDF5IO.h"
 
-//#define CCONVERTKNMIH5VOLSCAN_DEBUG
+// #define CCONVERTKNMIH5VOLSCAN_DEBUG
```

Comment:
Can you update the News.md, Dockerfile and Definitions.h with a version update?

Reply:
Done